electron.js

```diff
@@ -16,48 +17,50 @@ function createWindow() {
     icon: __dirname + '/data/icon.png',
     frame: false,
     // fullscreen: true
-  });
-  BrowserWindow.addDevToolsExtension(
-    `${require(
-      'os'
-    ).homedir()}/.config/chromium/Default/Extensions/lmhkpmbekcpmknklioeibfkpmmfibljd/2.15.1_0`
-  );
+  })
+  if (fs.existsSync(REDUX_DEV_TOOLS)) {
+    BrowserWindow.addDevToolsExtension(REDUX_DEV_TOOLS)
+  } else {
+    console.warn('Redux DevTools not found!')
+    console.log(REDUX_DEV_TOOLS)
```

Comment:
rimosso `console.log(REDUX_DEV_TOOLS)`
il warn lo lascerei in quanto ora con il controllo fa sì che ignori la mancanza del Dev Tools
TODO: un sistema multiplex che determina se è in esecuzione una versione produzione o development